README.md

```diff
@@ -13,29 +13,37 @@ npm i webostv-as-promised -S
 ### webOS API
 
 ```javascript
-import { WebOSPromised } from 'webostv-as-promised';
+import { promisifyWebOS } from 'webostv-as-promised';
 
-const webOS = new WebOSPromised;
+const promisedWebOS = promisifyWebOS(window.webOS);
 
-console.log(webOS.libVersion);
+console.log(promisedWebOS.libVersion);
 
-console.log(webOS.systemInfo());
+console.log(promisedWebOS.systemInfo());
 
-webOS.deviceInfo()
+promisedWebOS.deviceInfo()
   .then(info => console.log(info));
+
+const systemTimeRequest = promisedWebOS.service.request('luna://com.palm.systemservice', {
+    method: 'time/getSystemTime',
+    parameters: { subscribe: true }
+});
+systemTimeRequest.promise.then(res => console.log(res));
+// or
+systemTimeRequest.return.cancel();
```

Comment:
Maybe it's better to use `returnValue` or `result` instead of `return`

Reply:
`returnValue` 🔥